WooCommerce/WooCommerceTests/Extensions/DateWooTests.swift

```diff
@@ -4,81 +4,108 @@ import XCTest
+        let nineAgo = NSLocalizedString("Updated 9 hours ago",
+                                        comment: "A unit test string for a plural time interval in hours")
+        let almostDayAgo = NSLocalizedString("Updated 23 hours ago",
+                                             comment: "A unit test string for time interval just under 1 day")
+        let dayAgo = NSLocalizedString("Updated 24 hours ago",
+                                       comment: "A unit test string for 1 day, represented as plural time interval in hours")
+
         // 2 minutes
         let twoMinutesAgo = Calendar.current.date(byAdding: .minute, value: -2, to: Date())!
-        XCTAssertEqual(twoMinutesAgo.relativelyFormattedUpdateString, "Updated 2 minutes ago")
+
+        XCTAssertEqual(twoMinutesAgo.relativelyFormattedUpdateString, minutesAgo)
 
         // 2 minutes, 3 seconds
         let twoPlusMinutesAgo = Calendar.current.date(byAdding: .second, value: -123, to: Date())!
-        XCTAssertEqual(twoPlusMinutesAgo.relativelyFormattedUpdateString, "Updated 2 minutes ago")
+        XCTAssertEqual(twoPlusMinutesAgo.relativelyFormattedUpdateString, minutesAgo)
 
         // 59 minutes
         let twoFiftyNineMinutesAgo = Calendar.current.date(byAdding: .minute, value: -59, to: Date())!
-        XCTAssertEqual(twoFiftyNineMinutesAgo.relativelyFormattedUpdateString, "Updated 59 minutes ago")
+        XCTAssertEqual(twoFiftyNineMinutesAgo.relativelyFormattedUpdateString, almostHourAgo)
 
         // 1 hour
         let oneHourAgo = Calendar.current.date(byAdding: .hour, value: -1, to: Date())!
-        XCTAssertEqual(oneHourAgo.relativelyFormattedUpdateString, "Updated 1 hour ago")
+        XCTAssertEqual(oneHourAgo.relativelyFormattedUpdateString, hourAgo)
 
         /// 9 hours
         let nineHoursAgo = Calendar.current.date(byAdding: .hour, value: -9, to: Date())!
-        XCTAssertEqual(nineHoursAgo.relativelyFormattedUpdateString, "Updated 9 hours ago")
+        XCTAssertEqual(nineHoursAgo.relativelyFormattedUpdateString, nineAgo)
 
         /// 23 hours, 59 minutes
         let underOneDayAgo = Calendar.current.date(byAdding: .minute, value: -1439, to: Date())!
-        XCTAssertEqual(underOneDayAgo.relativelyFormattedUpdateString, "Updated 23 hours ago")
+        XCTAssertEqual(underOneDayAgo.relativelyFormattedUpdateString, almostDayAgo)
 
         // 1 day
         let oneDayAgo = Calendar.current.date(byAdding: .day, value: -1, to: Date())!
-        XCTAssertNotEqual(oneDayAgo.relativelyFormattedUpdateString, "Updated 24 hours ago")
+        XCTAssertNotEqual(oneDayAgo.relativelyFormattedUpdateString, dayAgo)
     }
 
     func testUpdateStringWorksForIntervalsOverOneDay() {
 
         // Skip verifying the time part of the resulting string because of TZ madness
 
+        // Create and translate the exact phrase for the expected result
+        let updatedOct = NSLocalizedString("Updated on Oct 10, 2018",
```

Comment:
similar to the comment above, this string isn't in `Localizable.strings` and thus not localizable. you can use `NSLocalizedString("Updated on %@", ...)` and `String.localizedStringWithFormat` to create the expected string